### PR TITLE
Add alternative installation using wget and sh

### PR DESCRIPTION
**Pull request by ryukinix**
To avoid confusions and other types of infinite lots of pain, like #5 .

You can just put this in one terminal (when this will be merged):

* `wget  -O - https://raw.githubusercontent.com/brendan-rius/jupyter-c-kernel/master/install.sh | sh`

### CODE REVIEW

File: install.sh [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+repo_name="jupyter-c-kernel"
+repository=git@github.com:brendan-rius/jupyter-c-kernel.git
+
+echo ":: Cloning Jupyter C-kernel... "
+git clone $repository $repo_name; echo "Done. "
+echo ":: Installing python module C kernel."
+sudo -H pip install $repo_name; echo "Done. "
+echo ":: Installing kernel specification"
+cd $repo_name
+sudo jupyter kernelspec install c_spec/ ; echo "Done."
```
Comment (brendan-rius) [inline install.sh]: It should probably be `jupyter-kernelspec` here.
At least, I don't have the `jupyter` command. Is that also your case? Does it depend on the version (I am on 4.2.2)?

**Comment by brendan-rius** [comment]
Thanks for your work, it is in fact simpler with your script.

There a little problem to fix before I can merge it, (I added a note in on of the commits:)[https://github.com/brendan-rius/jupyter-c-kernel/pull/6/files]

File: install.sh (hunk 1 above)
Comment (ryukinix) [inline install.sh]: Hmm, interesting. My main system broken and I don't know more which version of Jupyter I was using. But I'll change that. `jupyter` and `jupyter-kernelspec` exists on my machine.

**Comment by ryukinix** [comment]
Thanks! I'll change that.

**Comment by brendan-rius** [comment]
Thank you!

**Comment by brendan-rius** [comment]
a7bcd78fdcb3507976c28fe57cb1463fa1b9a073 removes the `sudo` in the script because it did not seem useful. Was there a special reason to add it or can I remove it safely? :)

**Pull request merged**